index.html

```diff
@@ -1,14 +1,223 @@
+      <a href="learn-more.html">
+        <div class="learn-more"></div>
+      </a>
+    </div>
+  </div>
+
+
+  </section>
+  <section id="footer">
+    <p id="form-title">QUESTIONS OR COMMENTS? <B>GET IN TOUCH:</B></p>
+
+    <form>
+      <input type="text" name="name" id="name" value="Name" />
+      <input type="text" name="email" id="email" value="E-mail" />
+      <br>
+      <textarea name="comments" id="comments" cols="50" rows="10" value="Message"></textarea><br>
+      <input onclick="popUp()" class="submit" type="submit" value="" />
+    </form>
+
+    <script>
+      function popUp() {
+       confirm("Thank You for Your Feedback")
+      }
+    </script>
+
+    <!-- Side form message -->
+    <div id="footer-div">
+      <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ut harum nihil dolores magni assumenda sequi ad
+        voluptatibus totam eligendi excepturi asperiores, quasi, aperiam fugit soluta quae molestiae accusantium ipsam
+        nesciunt exercitationem quos? Minima itaque esse doloribus nam cum ea dignissimos animi? Dolores, reprehenderit?
+      </p>
+    </div>
+
+    <!-- Contacs and socials plug -->
+    <div id="contact-links">
+      <div id="find-me" class="contact">
+        <img class="logo" id="home" src="\projects\remock-d-morg\Images\home.svg" />
+        <p class="media-tags">
+          13 Elm str.<br>
+          Camp Cristal lake, VT.<br>
+          05829
+      </div>
+      <div id="call-me" class="contact">
+        <img class="logo" src="\projects\remock-d-morg\Images\phone.svg" />
+        <a href="1-(867)-5309" class="media-tags" id="phone">1-(867)-5309</a>
+      </div>
+      <div id="mail-to" class="contact">
+        <img class="logo" src="\projects\remock-d-morg\Images\mail.svg" />
+        <a href="fakeemail@darkweb.com " class="media-tags">fakeemail@darkweb.net</a>
+      </div>
+    </div>
+    <div id="media-links">
+      <div id="facebook" class="contact">
+        <img class="logo2" src="\projects\remock-d-morg\Images\facebook.svg" />
+        <a class="media-tags">Follow me on FB!</a>
+      </div>
+      <div id="insta" class="contact">
+        <img class="logo2" src="\projects\remock-d-morg\Images\insta.svg" /><a class="media-tags">And insta!</a>
+      </div>
+      <div id="youtube" class="contact">
+        <img class="logo2" src="\projects\remock-d-morg\Images\youtube.svg" />
+        <a class="media-tags">SFW content at youtube!!</a>
+      </div>
+      <div id="twitter" class="contact">
+        <img class="logo2" src="\projects\remock-d-morg\Images\Twitter.svg" />
+        <a class="media-tags">Cant forget twatter!</a>
+      </div>
+    </div>
+
+
+    <!-- ----artist creds---- -->
+
+   <div id="credit">Icons sourced by from <a
+        href="https://www.freeicon.io" title="FreeIcon">www.freeicon.io</a>
+        <br>
+        Title icon sourced from <a href="https://seeklogo.com">www.seeklogo.com</a>
+      </div>

```

Comment:
Always good to give credit! Another thing you could do is add this to the images `alt="some text here"` or "alternate text". Another area is in the read me file of the program. Just alternative places that may help keep page styling cleaner and real estate free.